ensime-client.el

```diff
@@ -996,10 +996,9 @@ copies. All other objects are used unchanged. List must not contain cycles."
 (defun ensime-rpc-async-typecheck-files (file-names continue)
   (ensime-eval-async `(swank:typecheck-files ,file-names) continue))
 
-(defun ensime-rpc-async-typecheck-file-with-contents (file-name contents continue)
-  (if (version< (ensime-protocol-version) "0.8.11")
```

Comment:
we could probably remove a tonne more of these checks because everybody should be using the latest version of ensime-server now.